plugin/trace/setup.go

```diff
@@ -34,14 +34,30 @@ func setup(c *caddy.Controller) error {
 	return nil
 }
 
+func endPointName(hosts []string, port string) string {
```

Comment:
What does this do ?

Reply:
it build a representation of empty, 1 or multiple addresses attached to the same port ..
trace.ServiceEndpoint use at as a name.

can be 
:53
127.0.0.1:53
(127.0.0.1,::1):53

Reply:
not sure how to get better, but we need to distinguish each BlocList with is identified by Zone + bind addresses.

Reply:
Right now, used only by Trace plugin. But should be available for all ... it is the way to have an identifier of the Config once the server is started.

Reply:
ok, please comment this.

Reply:
I changed it : move to Config as HostAddresses() and return something a bit more usual ..